docs/docs/install-customize-cmd.mdx

```diff
@@ -20,7 +20,7 @@ Export-PoshTheme -FilePath ~/.mytheme.omp.json -Format json
 Once you're done editing, adjust your `$PROFILE` to use your newly created theme.
 
 ```powershell
-oh-my-posh --init --shell pwsh --config ~/.mytheme.omp.json | Invoke-Expression
+Set-PoshPrompt -Theme ~/.mytheme.omp.json
```

Comment:
That's not going to work. This guide is for those installing things via scoop/homebrew/winget. The powershell side contains the right info. If you've followed the guide per platform, doing this will effectively fail.

Reply:
right. I ran into the wrong section